### PR TITLE
docs: Cleanup module docs

### PR DESCRIPTION
**Pull request by ibgreen**
- Break out more react-graph-layers react components into modules/react
- Reorganize docs into consistent structure for each module (developer-guide/api-reference)
- Minimal npm READMEs (these always get out of sync otherwise).

### CODE REVIEW

File: docs/modules/bing-maps/developer-guide/get-started.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# deck.gl-bing-maps
+
+This module allows [deck.gl](https://deck.gl) to be used as a Bing Maps custom layer.
+
+## Installation
+
+```bash
+npm install deck.gl deck.gl-bing-maps
```
Comment (donmccurdy) [inline docs/modules/bing-maps/developer-guide/get-started.md:8]: The package.json shows `@deck.gl-community/bing-maps`, is this intentional?

File: docs/modules/bing-maps/developer-guide/get-started.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+## Usage
+
+```ts
+import {loadModules} from 'deck.gl-bing-maps';
```
Comment (donmccurdy) [inline docs/modules/bing-maps/developer-guide/get-started.md:14]: See above.

**Review by donmccurdy — APPROVED** [review]
I like the docs cleanup!

File: docs/modules/bing-maps/developer-guide/get-started.md (hunk 1 above)
Comment (ibgreen) [reply docs/modules/bing-maps/developer-guide/get-started.md:8]: The team should vote on that. Reverted for now.

**Pull request merged**